doc/code_examples/Tutorial_AASequence.cpp

```diff
@@ -30,6 +30,8 @@
 
 //! [AASequence]
 
+// This script calculates the mass-to-charge ratio of a 2+ charged b-ion and full peptide from a hardcoded sequence.
```

Comment:
it does a bit more but I think you mean that we should add more information at the top of the script

Reply:
Yes that's what I mean. I think this will help users understand what the script does and figure out if it's the right tool for them.